package.json

```
@@ -57,6 +58,12 @@
         "category": "ONE",
         "_comment": "Call this command to set the model directory of local machine."
       },
+      {
+        "command": "onevscode.show-compile-webview",
+        "title": "compile",
+        "category": "ONE",
+        "_comment": "call this when compiling *.tflite, *.onnx, *.pth, etc"
```

```suggestion
        "_comment": "Call this when compiling *.tflite, *.onnx, *.pb, etc"
```

Maybe pb instead of pth?

---

Do we support pth compilation? 

https://github.com/hyunsik-yoon/ONE-vscode/blob/2b8b2af0c37f469ca8c688da030cbb29027f9e43/src/Views/FileTreeView.ts#L374

---

For *.pth, I've heard that by word of mouth. 🐱 
I tried to find info myself and found that `one-import-pytorch` seems to be used only in test.

https://github.com/Samsung/ONE/blob/6921c7500a628d1f81c1d586ffcfd924e69e06a6/compiler/one-cmds/tests/pytorch-operations/CMakeLists.txt#L30-L31

@llFreetimell or @mhs4670go, could you share information about `pth` support if any?

---

AFAIK, we are supporting pytorch with https://github.com/Samsung/ONE/blob/6921c7500a628d1f81c1d586ffcfd924e69e06a6/compiler/one-cmds/one-import-pytorch...!
(But I am not sure is it official supported or unofficially supported...)

---

Thanks @llFreetimell 

It seems that it is an experimental feature. Document, help message, etc are not yet updated.

```
# pytorch importer is an experimental feature, it is not used in default configuration
if(ENABLE_ONE_IMPORT_PYTORCH)
  list(APPEND ONE_COMMAND_FILES one-import-pytorch)
endif(ENABLE_ONE_IMPORT_PYTORCH)
```

@hyunsik-yoon It's up to you :) Please go ahead if you think this feature will be supported in the release in September.